va/va_dec_hevc.h

```diff
@@ -327,7 +475,75 @@ typedef struct  _VASliceParameterBufferHEVC
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VASliceParameterBufferHEVC;
 
+/**
+ * \brief HEVC Extented Slice Parameter Buffer Structure For Long Format
+ *
+ * This data structure contains both baseline HEVC profiles (main, main10)
```

Comment:
is the structure below required for main and main10?

Reply:
@xhaihao no, they are required only for hevc range extension and scc decoding.